### PR TITLE
[Release-1.19] Upgrade nginx chart to version 4.0.305

### PR DESCRIPTION
**Pull request by galal-hussein**
#### Proposed Changes ####

Upgrade Nginx ingress to version 4.0.3

#### Types of Changes ####

chart upgrade

#### Verification ####

- Install rke2 cluster

#### Linked Issues ####

- https://github.com/rancher/rke2/issues/1940

### CODE REVIEW

File: scripts/build-images [hunk 1]
```diff
@@ -21,7 +21,7 @@ xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
     docker.io/rancher/klipper-helm:v0.4.3-build20210225
     docker.io/rancher/pause:3.2
     docker.io/rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1
```
Comment (brandond) [inline scripts/build-images:23]: Can drop this as it's not used with the newer chart.

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/build-images (hunk 1 above)
Comment (galal-hussein) [reply scripts/build-images:23]: its just disabled in the new chart but if the user decided to enable it still needs the image

Comment (brandond) [reply scripts/build-images:23]: We don't include it in 1.21 which is on the new-style chart, so either we need to re-add it there, or remove it from here.
https://github.com/rancher/rke2/blob/release-1.21/scripts/build-images#L24

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/build-images (hunk 1 above)
Comment (galal-hussein) [reply scripts/build-images:23]: yeah I think I will readd it in 1.21 and master, and will add webhook image here

**Review by brandond — APPROVED** [review]

**Pull request merged**